<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContinuedFractionCalculatorMainWindow</class>
 <widget class="QWidget" name="ContinuedFractionCalculatorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Continued Fraction Calculator</string>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true">Continued Fraction Calculator</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Expression</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="expression_line_edit">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Depth</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSlider" name="depth_slider">
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="depth_spin_box">
         <property name="maximum">
          <number>40</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Numerators</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="numerators_simple_radio_button">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Simple</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="numerators_customized_radio_button">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Customized</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Sequence</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_6">
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Cycle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="numerators_sequence_line_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="numerators_cycle_line_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="calculate_push_button">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Results</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="result_render_area">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Results will be rendered here</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>numerators_simple_radio_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>numerators_sequence_line_edit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numerators_simple_radio_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>numerators_cycle_line_edit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>depth_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>depth_spin_box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>depth_spin_box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>depth_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
